### DISCUSSION TITLE
Launcher extension based on Cinnamenu

### DISCUSSION THREAD

**Discussion by anaximeno** [linuxmint/discussions#592] [Ideas]
I think a good idea for an extension could be a Launcher, something like [Ulauncher](https://ulauncher.io/), or like the PopOS launcher:

![image](https://github.com/user-attachments/assets/80d9cb78-99de-4b34-ac8a-48f31c29047e)

For that reason, the applet [Cinnamenu](https://github.com/linuxmint/cinnamon-spices-applets/tree/master/Cinnamenu%40json) (and the default [Cinnamon Menu](https://github.com/linuxmint/cinnamon/tree/master/files/usr/share/cinnamon/applets/menu%40cinnamon.org)) could serve as a good base/example.

**Comment by Jopp-gh** [comment]
Totally. Mint's launcher is useful but too "clunky" and too windows-alike.

As ex Mac user I might be influenced by the design of spotlight (launcher on Mac) but to be honest, I think to like more the simple look of a search mask. To pull down/up some kind of dashboard with everything is just too much.

To launch commands and apps without the visual clutter of the original search panel in Mint is more relaxing and more focused. Plus, Ulauncher comes with pretty useful and nice extensions.

Why not rebuilding Mint's main search panel a la Ulauncher, with the ability to add extensions?

**Comment by Secret-chest** [comment]
I'd also like it to have content plugins. For example, searching `computer` might show the computer place, the 💻 character, Google searches about computers, Wikipedia article named computer and so on. Cinnamenu is nice but not extensible, something like Nemo actions would be nice.

**Reply by anaximeno** [reply]
Interesting, I'll consider some of these going to the future.

**Comment by Secret-chest** [comment]
And maybe Super could be changed to launch it instead of the menu by default - this is what most people use Super for. Of course it should be able to be changed back.

**Reply by Jopp-gh** [reply]
Well, the key to invoke menu@cinnamon is already configurable. 

Just Right click menu@cinnamon in your panel to open its options and set another keystroke.

**Comment by anaximeno** [comment]
Extension Proposal: https://github.com/linuxmint/cinnamon-spices-extensions/pull/1152
